src/backend/distributed/master/master_node_protocol.c

```diff
@@ -648,7 +648,7 @@ GetTableDDLEvents(Oid relationId)
 		{
 			isConstraint = true;
 		}
-		else if (indexForm->indisunique)
+		else if (indexForm->indisunique || indexForm->indisexclusion)
```

Comment:
Would be nice to mention exclusion alongside uniqueness in the comment section above the if statements